### DISCUSSION TITLE
Gotek for the A500 stuck at AUTOBOOT: T:00.0

### DISCUSSION THREAD

**Discussion by poroxius** [keirf/flashfloppy#850] [General]
Hi everyone. I've went through a lot of issues with my Gotek, but I finally solved them all (kinda). I bought a Gotek for my Amiga 500 a couple of months ago and I decided to mod my External Floppy Drive that's laying around into a external Gotek drive. I modded the Gotek with the OLED screen and I flashed it with FlashFloppy, I bought a DF0 to DF1 switcher and mounted it on my motherboard and so on, but now I'm finally over with the configuration. Unfortunately, I've configured the Gotek wrong so it fried one of my USBs, but I solved that by completely removing the +12V PSU connector from the Gotek and External Drive's PSU connector and I connected the +5V connector to the +5V pin on the Gotek. Now it isn't frying my USBs and the Gotek is working, but it's stuck at Autoboot: T:00.0. Idk what's happening, I've done everything, put the autoboot.hfe and hxcsdfe.cfg from the FlashFloppy v3.41 (with whom I flashed the Gotek) files onto the USB drive ([as mentioned in this tutorial that I followed](https://www.youtube.com/watch?v=InIReYzqHiw&ab_channel=Retro32)), but it's stuck. I switched the Amiga to the DF1 drive with the DF0 to DF1, but nothing happens, I'm just stuck at the Floppy Screen and the Gotek is at T:00.0. What did I do wrong?

**Comment by keirf** [comment]
The Amiga isn't able to recognise or talk to the external drive. I don't know why. Some wiring issue perhaps.

**Reply by poroxius** [reply]
I forgot to mention that I added the ff.cfg file with the interface = amiga line, but it still isn't working.

**Reply by keirf** [reply]
If the display doesn't change at all from T0.0 nor do you see the Gotek green light, then there is something else wrong. Eg perhaps drive select jumper is wrong for your switcher (could try S1 instead of S0).

**Comment by poroxius** [comment]
> If the display doesn't change at all from T0.0 nor do you see the Gotek green light, then there is something else wrong. Eg perhaps drive select jumper is wrong for your switcher (could try S1 instead of S0).

I switched the jumper from S1 to S0 and it worked! Now I booted FlashFloppy successfully and the green light is flashing! Thank you so much keirf! And thank you for your awesome work for FlashFloppy and the community!
